frontend/src/config/index.ts

```diff
@@ -1 +1,18 @@
+import mapErrorToString from '@/utils/map-error-to-string';
+import axios from 'axios';
+
+// Intercept the axios response to map errors to strings.
+axios.interceptors.response.use(
+  (r) => r,
+  (error) => {
+    console.error(error);
+
+    if (error instanceof Error) {
+      error.message = mapErrorToString(error);
+    }
```

Comment:
Do you think we really need this guard here?
What else could be passed to this error handler?
From the docs:
> Any status codes that falls outside the range of 2xx cause this function to trigger
https://axios-http.com/docs/interceptors

Reply:
Yes you're right :) I will remove the guard